ivy/functional/backends/numpy/experimental/linear_algebra.py

```diff
@@ -92,15 +94,18 @@ def kron(
 
 kron.support_native_out = False
 
-
+@with_supported_dtypes({"1.11.0 and below": ("float16", "float32", "float64", "complex64", "complex128")}, backend_version)
 def matrix_exp(
     x: np.ndarray,
-    /,
+    /,complex64
```

Comment:
Currently I can't see the mismatch error in the CI because of this error here. I guess you accidentally left this in the arguments.

Reply:
Uhh! ;) Sorry for that. Now tests are passing 😊.